tcms/utils/tests/test_github.py

```diff
@@ -8,6 +11,19 @@
 from tcms.utils import github
 
 
+class CalculateSignatureTestCase(test.TestCase):
+    def test_string_arguments(self):
+        with self.assertRaisesRegex(TypeError, 'key: expected bytes or bytearray, but got \'str\''):
+            github.calculate_signature('secret', 'content')
```

Comment:
that's not functionality coming from our own code base so no need to test it. This is part of hmac/hashib internals